src/update-unicode-tables.make

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env -S make -f
+
+all: arabic-table emoji-table indic-table tag-table ucd-table use-table vowel-constraints
+
+arabic-table: gen-arabic-table.py ArabicShaping.txt UnicodeData.txt Blocks.txt
+	./$^ > hb-ot-shape-complex-arabic-table.hh || (rm hb-ot-shape-complex-arabic-table.hh; false)
+emoji-table: gen-emoji-table.py emoji-data.txt
+	./$^ > hb-unicode-emoji-table.hh || (rm hb-unicode-emoji-table.hh; false)
+indic-table: gen-indic-table.py IndicSyllabicCategory.txt IndicPositionalCategory.txt Blocks.txt
+	./$^ > hb-ot-shape-complex-indic-table.cc || (rm hb-ot-shape-complex-indic-table.cc; false)
+tag-table: gen-tag-table.py languagetags language-subtag-registry
+	./$^ > hb-ot-tag-table.hh || (rm hb-ot-tag-table.hh; false)
+ucd-table: gen-ucd-table.py ucd.nounihan.grouped.zip hb-common.h
+	./$^ > hb-ucd-table.hh || (rm hb-ucd-table.hh; false)
+use-table: gen-use-table.py IndicSyllabicCategory.txt IndicPositionalCategory.txt UnicodeData.txt Blocks.txt
+	./$^ > hb-ot-shape-complex-use-table.cc || (rm hb-ot-shape-complex-use-table.cc; false)
+vowel-constraints: gen-vowel-constraints.py ms-use/IndicShapingInvalidCluster.txt Scripts.txt
+	./$^ > hb-ot-shape-complex-vowel-constraints.cc || (hb-ot-shape-complex-vowel-constraints.cc; false)
+
+packtab:
```

Comment:
Nothing depends on this target. It should be depended on by `emoji-table` and `ucd-table`.

Reply:
Breaks `./$^` however :/ Added it to `all`.